cmd/cli/root.go

```diff
@@ -80,3 +86,19 @@ func initConfig() {
 		cobra.CheckErr(viper.ReadInConfig())
 	}
 }
+
+func initTelemetry() {
+	// bail if this has been disabled.
+	if os.Getenv("PIRI_DISABLE_ANALYTICS") != "" {
+		return
+	}
```

Comment:
Will need to document this opt-out procedure, and also include a list of the data we collect when it's enabled.